libraries/GCS_MAVLink/GCS_Param.cpp

```diff
@@ -75,7 +75,7 @@ GCS_MAVLINK::queued_param_send()
     }
     count -= async_replies_sent_count;
 
-    while (count && _queued_parameter != nullptr && get_last_txbuf() > 50) {
+    while (count && _queued_parameter != nullptr && last_txbuf_is_greater(33)) {
```

Comment:
Maybe a `static constexpr` for the magic number?

Reply:
What would we call it?  one_third?  Is that really better?  Anything less obvious and I would be annoyed when reading the code to have to look up the constant.   I considered eliminating the parameter and hard code 33 in last_txbuf_is_greater(), but I wanted to allow for other threshold values, even though I doubt we will need them.

But I'll make the change if you let me know what I should call it.